### PR TITLE
Find and replace state_write_counter with reversible_write_counter

### PR DESCRIPTION
**Pull request by z2trillion**
This is a followup to https://github.com/appliedzkp/zkevm-circuits/pull/409

### CODE REVIEW

File: src/zkevm_specs/evm/execution/call.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def call(instruction: Instruction):
             (CallContextFieldTag.StackPointer, instruction.curr.stack_pointer + 6),
             (CallContextFieldTag.GasLeft, instruction.curr.gas_left - gas_cost - callee_gas_left),
             (CallContextFieldTag.MemorySize, next_memory_size),
-            (CallContextFieldTag.StateWriteCounter, instruction.curr.state_write_counter + 1),
+            (CallContextFieldTag.StateWriteCounter, instruction.curr.reversible_write_counter + 1),
```
Comment (han0110) [inline src/zkevm_specs/evm/execution/call.py:154]: This should also be renamed.

```suggestion
            (CallContextFieldTag.ReversibleWriteCounter, instruction.curr.reversible_write_counter + 1),
```

**Review by han0110 — COMMENTED** [review]
I also did the same thing in #100, but it's nice to do this in a separate PR. (btw for circuit, I updated the naming in appliedzkp/zkevm-circuits#419.)

File: src/zkevm_specs/evm/execution/call.py (hunk 1 above)
Comment (z2trillion) [reply src/zkevm_specs/evm/execution/call.py]: thanks, i've updated those.

**Comment by z2trillion** [comment]
> I also did the same thing in #100, but it's nice to do this in a separate PR. (btw for circuit, I updated the naming in [appliedzkp/zkevm-circuits#419](https://github.com/appliedzkp/zkevm-circuits/pull/419).)

Great minds think alike:

I am renaming state write -> reversible write in https://github.com/appliedzkp/zkevm-docs too.

**Review by han0110 — APPROVED** [review]
Ah I didn't see that you already planed to fix it in spec and in doc. All renaming look good to me now!

**Review by ed255 — APPROVED** [review]
LGTM! Thanks for the update!

**Pull request merged**